### PR TITLE
OD-710 [Fix] Ensure links are working in helpers

### PR DESCRIPTION
**Pull request by OleksandrLavrynenko**
@romanyosyfiv

OD-710 https://weboo.atlassian.net/browse/OD-710

## Description
**Problem**: Event listener didn't work outside of the instance function
**Solution**: Event listeners moved in the instance function

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: js/build.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-(function () {
+Fliplet.Widget.instance('list', function (data) {
+  var $container = $(this);
+
   $('.linked[data-list-item-id]').click(function (event) {
```
Comment (tonytlwu) [inline js/build.js]: The event listener attachment should be scoped to each widget instance.

```suggestion
  $container.find('.linked[data-list-item-id]').click(function (event) {
```

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/build.js (hunk 1 above)
Comment (OleksandrLavrynenko) [reply js/build.js]: @tonytlwu  $('.linked[data-list-item-id]') and $container.find('.linked[data-list-item-id]') equally return the array of elements with selector  .linked[data-list-item-id] and assign the event listener for them

Comment (tonytlwu) [reply js/build.js]: With `Fliplet.Widget.instance(name, fn)`, the `fn` function is called for every single instance of the component. If you have multiple instances of the component on the screen — try dropping in multiple instances on the same page — the `fn` function will be called multiple times (once for each instance).

If you don't scope the event listener to each instance, the event listener will be attached multiple times for each instance.

**Review by tonytlwu — APPROVED** [review]
@romanyosyfiv @inna-bieshulia This is ready for testing

Risk: Low
Affected: List instances with links
Verify: Links are working as expected

Squash and merge: Use PR title

**Pull request merged**